.travis.yml

```diff
@@ -189,6 +206,45 @@ matrix:
         - docker run -d --network host --name symbolicator us.gcr.io/sentryio/symbolicator:latest run
         - docker ps -a
 
+    - <<: *postgres_default
+      name: 'Sentry-Relay integration tests'
+      env: TEST_SUITE=relay-integration DB=postgres
+      services:
+        - docker
+        - memcached
+        - redis-server
+        - postgresql
+      before_install:
+        - *before_install_default
+        - docker pull us.gcr.io/sentryio/relay:latest # pull relay we'll run and kill it for each test
+
+        - > # clickhouse
+          docker run
+          --name sentry_clickhouse
+          -d --network host
+          -v  sentry_clickhouse:/var/lib/clickhouse
+          --ulimit nofile=262144:262144
+          -e CLICKHOUSE_CONFIG=/etc/clickhouse-server/config.xml
+          yandex/clickhouse-server:19.11
+
+        - > # snuba
+          docker run
+          --name sentry_snuba
+          -d --network host
+          -p 127.0.0.1:1218:1218/tcp
+          -e PYTHONUNBUFFERED=1
+          -e SNUBA_SETTINGS=docker
+          -e DEBUG=1
+          -e CLICKHOUSE_HOST=127.0.0.1
+          -e CLICKHOUSE_PORT=9000
+          -e CLICKHOUSE_HTTP_PORT=8123
+          -e REDIS_HOST=127.0.0.1
+          -e REDIS_PORT=6379
+          -e REDIS_DB=1
+          getsentry/snuba devserver --no-workers
```

Comment:
_nit_: There are a few settings we might not need compared to devservices:
- `PYTHONUNBUFFERED`, since we don't consume the logs
- `DEBUG`, since we don't consume the logs
- `REDIS*`, since those are implied by `SNUBA_SETTINGS=docker`.
- `CLICKHOUSE*`: looks like this is the new settings format and `CLICKHOUSE_SERVER` is deprecated. Let's change the snuba integration test to also use the new values.
- running `devserver --no-workers` is not required since we don't require the devserver, probably.

Also:
- The snuba test suite sets `SNUBA_SETTINGS=test`. However, I think it's better to test with the docker settings, since they look closer to devservices and prod and provide a more realistic test scenario.